src/freenet/clients/http/wizardsteps/SECURITY_NETWORK.java

```diff
@@ -117,9 +117,12 @@ public void getStep(HTTPRequest request, PageHelper helper) {
 	 */
 	private void securityLevelChoice(HTMLNode parent, SecurityLevels.NETWORK_THREAT_LEVEL level) {
 		HTMLNode input = parent.addChild("p").addChild("input",
-		        new String[] { "type", "name", "value" },
-		        new String[] { "radio", "security-levels.networkThreatLevel", level.name() });
-		input.addChild("b", WizardL10n.l10nSec("networkThreatLevel.name."+level));
+		        new String[] { "type", "name", "value", "id" },
+		        new String[] { "radio", "security-levels.networkThreatLevel", "security-levels.networkThreatLevel" + level.name() });
```

Comment:
it looks like the value level.name() got lost here.

Reply:
Oops, I had to read those lines quite a few times to finally spot it. Sorry about that, I pushed a fix.

Reply:
No problem — that’s what review is for. The way we create our pages sadly prevents the compiler from catching these errors, so we have to be extra-careful.